### DISCUSSION TITLE
Quick Chapters?

### DISCUSSION THREAD

**Discussion by seanap** [advplyr/audiobookshelf#2310] [Ideas]
The chapter lookup and setting feature works amazingly well.  I've been slowly fixing all my chapters, and I'm finding that I keep doing the same 7 clicks over and over again.  Is there a way to make this process less manual or faster?

Something even as simple as making the Lookup popup select the Search button so I can hit the enter key and save a click.

I was thinking that as long as the audiobook is within 99.x % of the audible length and the book is matched, selecting "Quick Chapter" would automatically do the chapter lookup and save. This "quick chapter" feature would same me a ton of time and clicks.

**Comment by advplyr** [comment]
There is a feature request open where we talked about how to build an auto match chapters feature. It seems doable if the book has an ASIN and we set some reasonable parameters

**Comment by Maxklos** [comment]
I currently habe the same Problem. Do you have a # for the feature request? Otherwise it would be nice to know how ABS sets Chapters, sometimes it works, some times don't.

**Comment by presswizards** [comment]
@advplyr Any progress on Quick Fix Chapters from ASIN? I have a lot of books that have 0 chapters, and it's super annoying having to click Add Chapters, Lookup, Search, Apply, Save repeatedly... it should be the same as Quick Match.
